### PR TITLE
Navbar

### LINKED ISSUE #16: Feat: Notification and profile modal on navbar
### Description

add Notification and profile modal on navbar

### Screenshots

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Somidh**
## Fixes Issue

Closes #16 

## Changes proposed

- Added navbar
- Added profile modal
- Added notification model

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
should be on left

image

too close to button, push it down a little 

Screenshot from 2023-02-25 05-32-36

File: app/components/NotificationIcon.tsx [hunk 1]
```diff
@@ -46,15 +46,13 @@ const NotificationIcon: FC = () => {
         MenuListProps={{
           "aria-labelledby": "basic-button",
         }}
-        className={` mt-4 ${open_sans.className}`}
+        className={` mt-4 -ml-24 ${open_sans.className}`}
```
Comment (Aadarsh805) [inline app/components/NotificationIcon.tsx:48]: there must be a position option on mui to display it on left or right

**Review by Aadarsh805 — APPROVED** [review]
LGTM

File: app/components/NotificationIcon.tsx (hunk 1 above)
Comment (Somidh) [reply app/components/NotificationIcon.tsx:48]: there is actually named anchorOrigin, didn't notice. Do u want me to change it or it's ok?

Comment (Aadarsh805) [reply app/components/NotificationIcon.tsx:48]: later

**Pull request merged**